### PR TITLE
*: remove useless code in executor.

### PR DESCRIPTION
**Pull request by hanfei1991**

### CODE REVIEW

File: distsql/distsql.go [hunk 1]
```diff
@@ -32,16 +32,14 @@ var (
...
-	_ PartialResult = &partialResult{}
+	_ NewSelectResult  = &newSelectResult{}
+	_ NewPartialResult = &newPartialResult{}
 )
 
-var selectResultGP = gp.New(2 * time.Minute)
-
-// SelectResult is an iterator of coprocessor partial results.
-type SelectResult interface {
+// NewSelectResult is an iterator of coprocessor partial results.
```
Comment (winoros) [inline distsql/distsql.go]: Why not just remove the `New` key word?

**Comment by coocood** [comment]
LGTM

File: distsql/distsql.go (hunk 1 above)
Comment (hanfei1991) [reply distsql/distsql.go]: it's too many things to change, do it in next pr....

Comment (winoros) [reply distsql/distsql.go]: ok

**Pull request merged**